components/Head.tsx

```diff
@@ -40,7 +40,8 @@ const Head = () => {
 export const PageHead = ({ subtitle }: { subtitle?: string }) => {
   return (
     <NextHead>
-      <title>Overture DMS{subtitle ? ` - ${subtitle}` : ''}</title>
+      {/* <title>COVID Cloud{subtitle ? ` - ${subtitle}` : ''}</title> */}
```

Comment:
Leaving this one here, in the back pocket, because I have a feeling they won't like not seeing the brand in the title (the favicon is pretty bad), and because if we added tabs, then the title will not be consistent.